projects/systelab-components/src/lib/image-viewer/image-viewer.component.spec.ts

```diff
@@ -177,5 +182,41 @@ describe('ImageViewerTestComponent', () => {
 		expect(imageViewerComponent.zoomScale.marks.length).toBeGreaterThanOrEqual(0);
 	});
 
+	it('should adjust image zoom on init', () => {
+		const imageViewerComponent = fixture.componentInstance.imageViewer;
+		fixture.componentInstance.setInitials();
+		expect(imageViewerComponent.imgParams.sliderZoomPct).toBeGreaterThanOrEqual(imageViewerComponent.sliderZoomMin);
+		expect(imageViewerComponent.imgParams.sliderZoomPct).toBeLessThanOrEqual(imageViewerComponent.sliderZoomMax);
+	});
+
+	it('should adjust image zoom after doing adjust', () => {
```

Comment:
Are we really testing that the image is adjusted? I don't see it. I think we need to check that the actual size of the image (width or height) is equal to the container element.